### DISCUSSION TITLE
Quick question: if symbol not supported in the exchange anymore

### DISCUSSION THREAD

**Discussion by HKby** [bmoscon/cryptofeed#996] [General]
1. symbol list I obtained from coingecko / coinmarketcap 
2. however the symbol may not supported in the exchange spot anymore, it always throw exceptions, how to mute it? Coz I dump over 500 symbols into different exchanges at same time, therefore, it's hard to remove symbol or exceptions for the time being, any idea or suggestions?

**Comment by jihun-yoon** [comment]
Could you please give us examples?

**Reply by HKby** [reply]
That's very long time ago, I will try to find out the original root cause and paste here

**Comment by bmoscon** [comment]
I cant tell if this is an at run time (i.e. when you first start the feedhandler, an exchange might complain a symbol is invalid) issue or a during runtime issue (a feed is running for days and suddenly a trading pair is no longer supported). if its at run time, it wouldnt be hard to write some code to take a list of symbols and validate it against legal symbols and just use the legal ones. Similarly, you can use the valid symbol list from the exchange (provided by the library) and just use that if you want "all symbols. For during run time,  You need to periodically check the valid symbols and stop feeds that have the bad symbols. Its not possible to do this gracefully, you'd have to kill async tasks (feeds) and start new ones
